ballerina-command/src/main/resources/cli-help/ballerina-dist.help

```diff
@@ -17,11 +17,13 @@ OPTIONS
 
 
 BALLERINA COMMANDS
-       Here is a list of available sub-commands:
-           list       List installed and available distributions
-           update     Update to the latest patch version of the active distribution
-           pull       Fetch a given distribution and set it as the active version
-           use        Make the specified distribution the active version
-           remove     Delete an installed distribution
-
-       See 'ballerina dist <command> -h' for more information on a specific command.
+        Here is a list of available subcommands:
+
+        update     Update to the latest patch version of the active distribution
+        pull       Fetch a distribution and set it as the active version
+        use        Mark a distribution as the active distribution
+        list       List local and available to download distributions
+        remove     Remove distributions in your local environment
+
+
+See 'ballerina dist <command> -h' for more information on a specific command.
```

Comment:
Should we have indentation here? Since this is under the commands section?

Reply:
This is not part of the command section. I think this indentation is correct